### PR TITLE
Update connect frontend/supply chain pages and remove hello world page in SDK docs - closes #548 

### LINKED ISSUE #548: Update Connect a Frontend/Suppy Chain pages and remove Hello World page in SDK docs
Connect a frontend: Say that we recommend React and put it first in the list..
 Tutorials: We say Supply Chain, but in the navigation Supply chain.
 Tutorials: Hello world - remove page

### PR DESCRIPTION
**Pull request by chris529**
Closes #548 

Parent issue #535

### CODE REVIEW

File: modules/ROOT/pages/guides/frontend.adoc [hunk 1]
```diff
@@ -21,12 +21,12 @@ NOTE: A **Lisk blockchain network** is a network of nodes that are built with th
 
 == Choose a framework
 
-Choose a framework based on personal preference.
-
 Some popular framework examples for developing apps are listed below:
 
-* {url_angular}[Angular^]
+However, it is recommended to use React to create a frontend.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/frontend.adoc]: I would remove this line here. We are generally framework agnostic.

File: modules/ROOT/pages/guides/frontend.adoc [hunk 2]
```diff
@@ -21,12 +21,12 @@ NOTE: A **Lisk blockchain network** is a network of nodes that are built with th
 
 == Choose a framework
 
-Choose a framework based on personal preference.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/frontend.adoc:25]: Keep this line, and add something like:

NOTE: Lisk uses React.js as framework of choice for the examples in the documentation

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please remove hello world tutorials also from navigation, and check if there are any remaining references go the tutorials that need to be removed as well.

File: modules/ROOT/pages/guides/frontend.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/guides/frontend.adoc]: OK, now done 👍

File: modules/ROOT/pages/guides/frontend.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/guides/frontend.adoc:25]: Excellent idea Mona...now done 👍

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 3]
```diff
@@ -87,7 +87,7 @@ console.log(tx.stringify());
 Custom transaction::
 +
 --
-The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
+The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World guide] is used as an example here.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 4]
```diff
@@ -182,7 +182,7 @@ console.log(tx.stringify());
...
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 5]
```diff
@@ -302,7 +302,7 @@ Replace `http://localhost:4000` with the url of the node, to which you want to b
 Custom Transaction::
 +
 --
-The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
+The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World guide] is used as an example here.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

File: modules/ROOT/pages/tutorials/index.adoc [hunk 6]
```diff
@@ -5,7 +5,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-sdk/guides.html
 :page-previous-title: Guides
 
-:url_hello: tutorials/hello-world.adoc
+:url_hello: guides/index.adoc#_the_hello_world_app
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/index.adoc]: We should not link to the guides index here, as the tutorial got removed, so should the link here be removed.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/guides/broadcast.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/guides/broadcast.adoc]: done...suggestion committed.

File: modules/ROOT/pages/tutorials/index.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/index.adoc]: Ok, now done 👍

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please set a custom anchor id for the hello world app on the `guides/index.adoc` page, to follow the style guide.

**Review by Tschakki — CHANGES_REQUESTED** [review]
And also please remove the whole table row of hello world in the tutorials index .

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 7]
```diff
@@ -87,7 +87,7 @@ console.log(tx.stringify());
 Custom transaction::
 +
 --
-The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
+The `HelloTransaction` from the xref:{url_guides_hello}[Hello World App] is used as an example here.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 8]
```diff
@@ -182,7 +182,7 @@ console.log(tx.stringify());
...
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 9]
```diff
@@ -302,7 +302,7 @@ Replace `http://localhost:4000` with the url of the node, to which you want to b
 Custom Transaction::
 +
 --
-The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World tutorials] is used as an example here.
+The `HelloTransaction` from the xref:{url_guides_hello}[Hello World App] is used as an example here.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]:
```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Comment by chris529** [comment]
Hi Mona, the 2 requested changes above should now be done.. :-)

**Review by Tschakki — CHANGES_REQUESTED** [review]
Ok great! Please also fix the existing conflicts.

File: modules/ROOT/pages/guides/broadcast.adoc [hunk 10]
```diff
@@ -5,7 +5,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_lisk_wallet: https://lisk.io/wallet
 
 :url_api: reference/api.adoc
-:url_tutorials_hello: tutorials/hello-world.adoc
+:url_guides_hello: guides/index.adoc#_the_hello_world_app
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/broadcast.adoc]: Please create a custom ID and use it here

**Review by Tschakki — CHANGES_REQUESTED** [review]
OK great! Just one minor issue

File: modules/ROOT/pages/guides/broadcast.adoc (hunk 10 above)
Comment (chris529) [reply modules/ROOT/pages/guides/broadcast.adoc]: OK done..

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please fix the conflicts :)

**Comment by chris529** [comment]
Conflicts should be resolved now :-)

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**